src/main/scala/com/itechart/ny_accidents/parse/AccidentsParser.scala

```diff
@@ -73,33 +74,48 @@ class AccidentsParser {
       toInt(accident, MOTORIST_INJURED),
       toInt(accident, MOTORIST_KILLED),
       toStringList(accident, CONTRIBUTING_FACTOR_VEHICLE_COLUMNS),
-      toStringList(accident, VEHICLE_TYPE_CODE_COLUMNS))
+      toStringList(accident, VEHICLE_TYPE_CODE_COLUMNS)
+    )
   }
 
-  private def toLong(accident: Row, column: Int): Option[Long] = {
-    Try(accident.getString(column).toLong).toOption
+
+  private def toLongFromInt(accident: Row, column: Int): Option[Long] = {
```

Comment:
Does that sound like a general shared function that could be moved outside of AccidentsParser?

Reply:
yes